### DISCUSSION TITLE
Quasar.dev website is not very fast

### DISCUSSION THREAD

**Discussion by kostyfisik** [quasarframework/quasar#11516] [Ideas / Proposals]
As far as I understand quasar.dev is build with Quasar. While it is claimed to be fast,  Google's PageSpeed is showing mediocre rating, see https://pagespeed.web.dev/report?url=https%3A%2F%2Fquasar.dev%2F 

Any examples of websites build with Quasar and with high score at PageSpeed?

Putting some effort into optimizing the main Quasar website can be a good idea, otherwise the claim about speed is not very clear...

![Screenshot from 2021-11-29 18-34-34](https://user-images.githubusercontent.com/5239997/143896988-c8a1b8c3-4654-46ee-a0be-f0d690c91091.png)

![Screenshot from 2021-11-29 18-34-26](https://user-images.githubusercontent.com/5239997/143896990-8ed7cf31-4d37-4ace-9553-2d24fb53736c.png)

**Comment by pdanpdan** [comment]
You do realize it is the documentation site, with multiple examples for all components (and it also caches everything for offline usage) - it is not a 1 page site tweaked for hit and run money grabbing :)

**Reply by kostyfisik** [reply]
Thats why I was asking about examples.  I took one project from quasar-awesome, https://pagespeed.web.dev/report?url=https%3A%2F%2Fwww.coloban.com%2F , however it is still 30-mobile\75 desktop.

**Reply by smolinari** [reply]
Have a look at google's and youtube's results. It's not the speed that is terribly important, but rather user experience. Yes, in some cases faster page speed results will get you better rankings faster, but if you are working on anything serious, Google will rank you well enough and page speed becomes less critical. 

Scott

**Reply by kostyfisik** [reply]
Ok, may you someone can add some basic instructions, how to make quasar-based website to become blazing fast? Good UX is good. E.g. into "Options and helpers" section.

**Reply by hawkeye64** [reply]
There is a discussion on the Discord forums about Lighthouse and making improvements to the score. You can find it here: https://discord.com/channels/415874313728688138/856517299816497173/912290470347018281

**Reply by pikil** [reply]
Just worth mentioning that while app.coloban.com itself runs on Quasar V2 in the present days, www.coloban.com (which is not the app itself) is still sitting on Quasar V1. The numbers I see on my side are a bit higher, btw :) Performance: 48 (mobile) / 78 (desktop)

You can see that Google search page has got 56 % in mobile performance, for example - so, it does really depend on many factors
https://pagespeed.web.dev/report?url=https%3A%2F%2Fwww.google.com%2Fsearch%3Fq%3Dnever%2Bgonna

But that is a good point in any case - every remark which helps identifying problems and help make quasar better in the end are awesome!

**Comment by kostyfisik** [comment]
@hawkeye64 
This doesn't seem to be public information, see the screenshot...
![Screenshot from 2021-11-30 22-04-48](https://user-images.githubusercontent.com/5239997/144111466-ed962f73-aa49-48d6-b37d-f4f9caf32bf3.png)

**Reply by smolinari** [reply]
@kostyfisik - You need to join Discord and the Quasar Server. Here is an invite: https://discord.gg/FNB4jquG

Scott

**Reply by hawkeye64** [reply]
Correct. If you are not a member of the server, you won't see anything.
